Gemfile

```diff
@@ -85,7 +85,7 @@ gem "mime-types",                       "~>3.0",             :require => false,
 
 # Modified gems (forked on Github)
 gem "handsoap",                         "=0.2.5.5",          :require => false, :source => "https://rubygems.manageiq.org" # for manageiq-gems-pending only
-gem "ruport",                           "=1.7.0.3",                             :source => "https://rubygems.manageiq.org"
+gem "ruport",                           "~>1.8.0"
```

Comment:
If this is no longer modified we should move it into the "regular gems" section.